samri/pipelines/diagnostics.py

```diff
@@ -170,6 +170,13 @@ def diagnose(measurements_base,
 			(realigner, melodic, [('out_file', 'in_files')]),
 			])
 
+        elif realign == "time":
```

Comment:
Here and on many other rows you seem to have indented with spaces instead of tabs. Please always indent with tabs.